.github/workflows/ci-lambda.yml

```
@@ -0,0 +1,98 @@
+name: CI - Python
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
```

nit: I'm not sure if this is necessary. I think we should opt for less filtering on these triggers unless specifying `types` is absolutely necessary:

```suggestion
  pull_request:
```

---

Not a big deal but I tend to agree. We don't do any filtering here in our other workflows and I don't think we need to add this prematurely.